.github/workflows/pr_build.yml

```diff
@@ -156,7 +156,7 @@ jobs:
           sudo apt install -y ${{ matrix.apt-dependencies }} || exit 1
         else # macOS
           echo "Downloading MacOSX10.12 SDK.."
-          curl --progress-bar -L -o macOS_SDK10.12.4.tar.xz "https://cloudflare-ipfs.com/ipfs/QmdNLMCUCw8hBTqinkc7ke88WgRb7M2mWQbccQPDsyaub8?filename=macOS_SDK10.12.4.tar.xz" || exit 1
+          curl --progress-bar -L -o macOS_SDK10.12.4.tar.xz "https://data.moto9000.moe/vircadia_packages/macOS_SDK10.12.4.tar.xz" || exit 1
```

Comment:
committed to wrong PR?

Reply:
This is so this actually builds.

Reply:
can we host it somewhere centralized at least?

Reply:
We are technically not allowed to share this and Kalila wanted to host it via this IPFS gateway. https://github.com/vircadia/vircadia/pull/1411#issuecomment-958132177
Since the gateway has shown to be extremely unreliable I just put it on my server.